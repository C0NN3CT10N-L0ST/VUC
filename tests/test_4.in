e
LPI is cool
q