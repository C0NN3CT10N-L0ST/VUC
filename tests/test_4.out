 ____________________________ 
|  __      ___    _  _____   |
|  \ \    / / |  | |/ ____|  |
|   \ \  / /| |  | | |       |
|    \ \/ / | |  | | |       |
|     \  /  | |__| | |____   |
|      \/    \____/ \_____|  |
|                            |
|       Encoder/Decoder      |
|  ------------------------  |
|         |Options|          |
|         e - encode         |
|         d - decode         |
|          q - quit          |
|____________________________|
Choose option:
> 
Enter text to encode (MAX length is 300 characters):

Encoded text
-----------------------------
SFX xj occs
-----------------------------
 ____________________________ 
|  __      ___    _  _____   |
|  \ \    / / |  | |/ ____|  |
|   \ \  / /| |  | | |       |
|    \ \/ / | |  | | |       |
|     \  /  | |__| | |____   |
|      \/    \____/ \_____|  |
|                            |
|       Encoder/Decoder      |
|  ------------------------  |
|         |Options|          |
|         e - encode         |
|         d - decode         |
|          q - quit          |
|____________________________|
Choose option:
> 
Exiting program...